ReactAndroid/src/main/java/com/facebook/react/views/image/MultiPostprocessor.java

```diff
@@ -52,9 +52,20 @@ public String getName() {
   public CacheKey getPostprocessorCacheKey() {
     LinkedList<CacheKey> keys = new LinkedList<>();
     for (Postprocessor p : mPostprocessors) {
-      keys.push(p.getPostprocessorCacheKey());
+      CacheKey cacheKey = p.getPostprocessorCacheKey();
+      if (cacheKey != null) {
+        keys.push(cacheKey);
+      }
+    }
+
+    switch (postprocessors.size()) {
```

Comment:
can you double check this method?
where is postprocessors defined? maybe you need to use keys instead?